src/planscape/plan/views.py

```diff
@@ -1,28 +1,25 @@
+from planscape import settings
```

Comment:
Nit: please run "Sort Imports"; the datetime and json imports should come first, I think.

Reply:
ok - done.
I've also filed #431 .